code/controllers/subsystem/map_vote.dm

```diff
@@ -53,7 +53,7 @@ SUBSYSTEM_DEF(map_vote)
 	last_message_at = world.time
 
 	var/list/messages = args.Copy()
-	to_chat(world, span_purple(boxed_message("Map Vote\n<hr>[messages.Join("\n")]")))
+	to_chat(world, span_purple(boxed_message("Map Vote<br><hr>[messages.Join("<br>")]")))
```

Comment:
```suggestion
	to_chat(world, span_purple(boxed_message("Map Vote<br><hr>[jointext(messages, "<br>")]")))
```
i'm pretty sure jointext is better for some byond reason. it's both faster and also some string tree stuff is avoided? It was Kapu or someone that was telling me about this a lifetime ago. i forget the specifics but i like the formatting of it anyways. ultimately your choice though